python/ray/tune/logger.py

```diff
@@ -122,6 +123,12 @@ def flush(self):
         self._log_syncer.sync_now(force=True)
         self._log_syncer.wait()
 
+    def update_location(self, worker_ip):
```

Comment:
update_location() sounds more harmless than this is, maybe sync_results_to_new_location()?